apps/web/nuxt.config.ts

```diff
@@ -58,7 +58,7 @@ export default defineNuxtConfig({
       storename: process.env.STORENAME || 'PLENTYSYSTEMS AG',
       noCache: process.env.NO_CACHE || '',
       configId: process.env.CONFIG_ID || '',
-      isHero: process.env.IS_HERO === 'true',
+      isHero: process.env.IS_HERO !== 'false',
```

Comment:
```suggestion
      isHero: true,
```
Is there any reason not to just set it to `true`?

Reply:
Not really.